webv2/api/rules_test.go

```diff
@@ -18,7 +18,6 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-
```

Comment:
nit: we should remove this?

Reply:
Didn't get you, what should we remove?

Reply:
You've accidentally removed a line here and the whole file is now added to CL

Reply:
Ahh gotcha thanks, will revert the change